src/lib/Libifl/tm.c

```diff
@@ -502,11 +498,10 @@ localmom()
 				case ECONNREFUSED:
 #ifdef WIN32
 				case WSAEINTR:
```

Comment:
I think we should put `#define` for this one to EINTR in portability.h and remove this case block

Reply:
Its not a straight forward change as we need to carefully study the repercussions of doing such a change. So not touching it in this PR. may be we should handle in a future PR.

Reply:
Currently there is only 9 place we are using WSAEINTR in whole code base and everywhere its doing exactly same stuff as EINTR. So I think its pretty straight forward change which will remove few more ifdef.

Reply:
Its not the number of places that I'm concerned off, but how best we can make the changes around it as platform independent, Its easy to textually change 9 lines but we don't know what impact it has on compilation and run time on different platforms. So lets go without it now and I promise to make this change in a proper way in an upcoming PR. If you are really insisting this change in the current PR for it to get your approval, please let me know why you think so that it has to be in the current PR.

Thank you in advance for your understanding

Reply:
If you making changes in next PR already, I am fine with going without it in this PR.